Makefile

```diff
@@ -52,16 +53,19 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen kustomize ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	$(KUSTOMIZE) build config/kustomize-to-helm/overlays/crds -o charts/moco/crds
+	$(KUSTOMIZE) build config/crd -o config/crd/tests
```

Comment:
Is `config/crd/tests` a directory for temporary files?
If so, please ignore this directory by `.gitignore`.

Reply:
I wanted to enable conversion webhooks in controller tests, so I used manifests with a patch for testing.
https://github.com/cybozu-go/moco/blob/693d9edb720b30e7d012c3319a71d66a8b1963d1/controllers/suite_test.go#L67

But, I have rewritten all controller tests to use v1beta2, so it may not be necessary now.
I will try to clean up the code.

Reply:
At the time of this pull request, the storage version was v1beta1, so I needed to run the conversion webhook on controller-test as well.

Reply:
added comments: https://github.com/cybozu-go/moco/pull/350/commits/b22d8b0174df63c8a7c193c24836318e78ec49bb